### PR TITLE
 chore: Add `/data/sql/custom` to gitignore

### PR DESCRIPTION
**Pull request by michaeldelago**
## Changes Proposed:
-  Add `/data/sql/custom/*` to gitignore

## Issues Addressed:
- Local additions to /data/sql/custom may get committed in
- Custom or local changes shouldn't make it into git

### CODE REVIEW

**Comment by Nyeriah** [comment]
If you're maintaining your own repository on GitHub or somewhere else you may want your custom changes commited. In this case I believe it's easier for experienced users to add the gitignore than for unexperienced ones to remove it

**Comment by michaeldelago** [comment]
I'm of the opinion that if a fork wants to push something ignored by git, that responsibility should be on the maintainer of that fork rather than the original repository.

Additionally, other potentially custom content is explicitly ignored by git as well

https://github.com/azerothcore/azerothcore-wotlk/blob/master/.gitignore#L17-L28

**Comment by Kitzunu** [comment]
I am in favor of this. 

[src](https://github.com/azerothcore/azerothcore-wotlk/tree/master/src)/[server](https://github.com/azerothcore/azerothcore-wotlk/tree/master/src/server)/[scripts](https://github.com/azerothcore/azerothcore-wotlk/tree/master/src/server/scripts)/Custom/

Could also be added

**Comment by michaeldelago** [comment]
@Kitzunu `src/server/scripts/Custom` (and un-ignoring appropriate files) has been added.

File: .gitignore [hunk 1]
```diff
@@ -29,6 +29,10 @@
 /.env*
 /apps/joiner
 /deps/deno
+/data/sql/custom/*
+/src/server/scripts/Custom/*
+!/src/server/scripts/Custom/custom_script_loader.cpp
```
Comment (acidmanifesto) [inline .gitignore:34]: this i am not feeling right about with the script loader.

Comment (michaeldelago) [reply .gitignore:34]: Good eye, I thought it was static. I'll be able to get that fixed in a little bit

Comment (michaeldelago) [reply .gitignore]: @acidmanifesto This should be fixed.

**Comment by acidmanifesto** [comment]
Will mark as needing feed back from other devs for their opinions on this.

**Comment by Maelthyrr** [comment]
Look fine for some staff members so I pass in "To be merged"

**Review by Maelthyrr — APPROVED** [review]

**Pull request merged**